starter_code/views/beers.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="beerItem">
+
+  {{#each beersList}}
+  <div>
+  <img id="item" src="{{this.image_url}}" alt="Image of a beer">
+    {{> oneBeer this}}
+  </div>
```

Comment:
le `this` à l'intérieur de votre partial ne sert à rien, et je suis même surpris qu'il ne crée pas d'erreur ? Idem pour celui dans randomBeer